packages/cogify/src/cogify/cli/cli.cover.ts

```diff
@@ -0,0 +1,83 @@
+import { ConfigProviderMemory } from '@basemaps/config';
+import { initConfigFromPaths } from '@basemaps/config/build/json/tiff.config.js';
+import { GoogleTms, Nztm2000QuadTms, TileId } from '@basemaps/geo';
+import { fsa } from '@basemaps/shared';
+import { CliId, CliInfo } from '@basemaps/shared/build/cli/info.js';
+import { Metrics } from '@linzjs/metrics';
+import { command, number, option, optional, restPositionals, string } from 'cmd-ts';
+import { CutlineOptimizer } from '../../cutline.js';
+import { getLogger, logArguments } from '../../log.js';
+import { createTileCover, TileCoverContext } from '../../tile.cover.js';
+
+const SupportedTileMatrix = [GoogleTms, Nztm2000QuadTms];
+
+export const BasemapsCogifyCoverCommand = command({
+  name: 'cogify-cover',
+  version: CliInfo.version,
+  description: 'Create a COG from a covering configuration',
```

Comment:
I think this description is for cogify-create ?

Reply:
You are correct, this is the wrong description, ill fix it